docs/rag.md

```diff
@@ -111,6 +158,21 @@ configureGenkit({
     ]),
   ],
 });
+```
+
+### Define an Indexer
+
+The following example shows how to create an indexer to ingest a collection of PDF documents
+and store them in a local vector database.
+
+It uses the local file-based vector similarity retriever
+that Genkit provides out-of-the box for simple testing and prototyping (_do not
+use in production_)
+
+#### Create the indexer and chunking config
```

Comment:
Let's make the chunking thing its own section since it's an extra-Genkit step. Give it an intro like "This example uses the llm-chunk library to ... The following definition configures it to ..."

Reply:
done. WDYT?